src/index.html

```diff
@@ -1,13 +1,70 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+     rel="preconnect"
+     href="https://fonts.gstatic.com" 
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <main class="main">
+      <div class="product-card">
+        <img 
+        src="./images/imac.jpeg" 
+        alt="MacBoock" 
+        class="mac-book"
+        >
+        <div class="product-card__text-box">
+          <h1 class="product-card__header">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+        </div>
+        <p class="product-card__code">Product code: 195434</p>
+        <div class="product-card__star-row">
+          <div class="product-card__star-item">
+            <div class="product-card__star-box">
+              <img 
```

Comment:
Reuse stars block from [Stars task](https://github.com/mate-academy/layout_stars) and keep it operational.